src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -632,7 +633,7 @@ private function addService($id, Definition $definition)
         }
 
         if ($definition->isAutowired()) {
-            $doc = <<<EOF
+            $doc = <<<'EOF'
```

Comment:
Should be reverted. I know that it comes from php-cs-fixer, but we should instead disable this rule in `.php.cs.dist`